src/main/java/com/datatorrent/netlet/DefaultEventLoop.java

```diff
@@ -513,14 +512,18 @@ public String toString()
 
             });
           }
-        }
-        catch (IOException ie) {
-          l.handleException(ie, DefaultEventLoop.this);
+        } catch (UnresolvedAddressException e) {
+          logger.error("Inet Address {} is not resolvable.", address, e);
```

Comment:
No, the goal of logging UnresolvedAddressException is to log unresolvable InetAddress along with the exception, UncaughtExceptionHandler will not help as it is generic "catch all" case and it was introduced to log exceptions raised on the DefaultEventLoop to the log file instead of System.stderr where default UncaughtExceptionHandler outputs them. Additionally, we do want to route UnresolvedAddressException to AbstractClient.handleException, so it will not be propagated to UncaughtExceptionHandler.

I don't see how DTLoggerFactory will help, as we do want those exceptions to be logged by default and do not want to set "com.datatorrent.netlet" to trace level for all customer deployments.